src/modules/helpchan.ts

```diff
@@ -39,6 +42,20 @@ This channel will be dedicated to answering your question only. Others will try
 For more tips, check out StackOverflow's guide on **[asking good questions](https://stackoverflow.com/help/how-to-ask)**.
 `;
 
+const occupiedMessage = (asker: User) => `
+**This channel is claimed by ${asker.toString()}.**
+It is dedicated to answering their questions only.
```

Comment:
A little late on this, but maybe
```suggestion
It is dedicated to answering their questions only; see <#${askHelpChannelId}> to claim your own.
```

Reply:
Added just "More info: <#${askHelpChannelId}>", so it will fit on one line. Two lines distracts a little from the "This channel is claimed by" message on the line above, in my opinion (fixable by adding a newline, but then the message starts to become longer than I think it should be.